knowledge_base/VERSIONING_ISSUES.md

```diff
@@ -0,0 +1,23 @@
+# Versioning Issues
+
+## Pytorch version
+This issue is related to https://github.com/pytorch/rl/issues/689. Using PyTorch versions lower < 1.13 and installing stable package leads to the following error:
```

Comment:
I guess that in the future we could have other undefined symbol, maybe we want to smooth things a bit (e.g.
> ```
> undefined symbol _ZN8pybind116detail11type_casterIN2at6TensorEvE4loadENS_6handleEb
> ```
> or another symbol

)

Reply:
Rephrased to make it clear that such specific error is just an example